src/widgets/scroll_zoom.rs

```diff
@@ -119,13 +107,14 @@ impl<T: Widget<EditorState>> ScrollZoom<T> {
 
     fn handle_zoom_cmd(&mut self, cmd: &Command, view_size: Size, data: &mut EditorState) {
         use crate::consts::cmd;
+        const ZOOM_DELTA: Vec2 = Vec2::new(50.0, 0.0);
```

Comment:
FWIW, I find this way too small. I don't fully understand the logic of wheel_zoom, but I note that Glyphs is approximately sqrt(2) each step (while adjusting to nice round numbers) and Chrome is approximately 2^(1/5). But refining the zoom logic (including a more useful center) should probably be separate than the Druid update.